### PR TITLE
add tests for categories_helper / resolves #1184

### LINKED ISSUE #1184: Adding missing tests for MoodsHelper
# Description 

Add missing tests! Use as an opportunity to refactor code (including getting rid of Rubocop disable comments like `# rubocop:disable MethodLength`)

# Do you want to be the assignee to work on this?

🚫

### PR DESCRIPTION
**Pull request by Veith3n**
# Description 
Add tests for categories_helper with little code refactoring
## Corresponding Issue
[#1184](https://github.com/ifmeorg/ifme/issues/1184#issue-370871368)
# Test Coverage
✅

### CODE REVIEW

File: spec/helpers/categories_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+describe CategoriesHelper do
+  subject { TestClass.new(build_stubbed(:category, id: 999)) }
```
Comment (julianguyen) [inline spec/helpers/categories_helper_spec.rb]: Why not just do `build_stubbed(:category)`?

File: spec/helpers/categories_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  class TestClass
```
Comment (julianguyen) [inline spec/helpers/categories_helper_spec.rb]: Why is this TestClass necessary?

File: spec/helpers/categories_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  describe '#quick_create_category_props' do
+    it 'call method with correct params' do
+      expect(subject).to receive(:quick_create_form_props).with(category_form_inputs_res, '/categories/quick_create')
+      subject.quick_create_category_props
+    end
+  end
+
+  describe '#edit_category_props' do
+    it 'call method with correct params' do
+      expect(subject).to receive(:edit_form_props).with(category_form_inputs_res, '/categories/999')
```
Comment (julianguyen) [inline spec/helpers/categories_helper_spec.rb]: It would be better to get the category id from `category.id` from your stubbed data

**Comment by hmasila** [comment]
This PR is similar to https://github.com/ifmeorg/ifme/pull/1194

**Pull request closed without merging**